source/SchemeEditor/TransitionItem.cpp

```diff
@@ -126,7 +126,7 @@ Energy TransitionItem::to() const
   return transition_.to();
 }
 
-void TransitionItem::updateArrow(double arrowDestY, double max_intensity)
+void TransitionItem::updateArrow(double arrowDestY /*, double max_intensity*/)
```

Comment:
Instead of commenting out, maybe use this?:
https://en.cppreference.com/w/cpp/language/attributes/maybe_unused